### ISSUE TITLE
Latency Issues

### ISSUE THREAD

**Issue by xHybred** [mausimus/ShaderGlass#69]
When using this in a game when I swapped from Shader Glass back to said game I noticed there was added input delay, I do not like this and I'm wondering if this is an issue you are aware of and is it possible to mitigate or reduce in future versions?

I'm a latency sensitive person

**Comment by Mistawani** [comment]
Yep, this app is awesome, but sadly the input lag added prevents me from using it :(

**Comment by mausimus** [comment]
There will be some lag due to Windows capture and then processing, the best you can do is make sure you have Output / Frame Skip = none.

**Comment by xHybred** [comment]
I understand that. What MS capture method are you using? DXGI?

Lossless Scaling also uses these capture services but it feels more responsive than this app.

But it does allow you to tune things such as "Allow Tearing" which helps with latency and queued frames, etc

**Comment by mausimus** [comment]
ShaderGlass uses WGC. I've been testing latency recently and will be adding tearing support in the next version, it does reduce it.

**Comment by mausimus** [comment]
Yes it should at least on Win 10, there is unfortunately a minor bug related to it (window gets misaligned, you have to move it to snap back) which I reported to Microsoft but looks unlikely to get fixed with Win 10 going out of service, so I left BitBlt as default. In my testing on Win 11 there's only one frame of latency in both modes (lowest I can do with Capture).
